### PR TITLE
Add to winget

### LINKED ISSUE #1: Add trh.exe to winget
Add the trh.exe to [winget](https://github.com/microsoft/winget-cli) for easy deployment on Windows.

How to can be found here: https://github.com/microsoft/winget-pkgs

### PR DESCRIPTION
**Pull request by secana**
Resolved #1 

@pkemkes when you have time, let's review the changes together.

### CODE REVIEW

File: src/Installer/Installer.wixproj [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    <Error Text="The WiX Toolset v3.11 (or newer) build tools must be installed to build this project. To download the WiX Toolset, see http://wixtoolset.org/releases/" />
+  </Target>
+  <!--
+	To modify your build process, add your task inside one of the targets below and uncomment it.
+	Other similar extension points exist, see Wix.targets.
+	<Target Name="BeforeBuild">
+	</Target>
+	<Target Name="AfterBuild">
+	</Target>
+	-->
```
Comment (pkemkes) [inline src/Installer/Installer.wixproj]: nit: Do you want to keep this comment? Looks to me like a standard template comment that could be removed.

File: README.md [hunk 2]
```diff
@@ -8,10 +8,26 @@ CLI tool to compute the [TypeRefHash (TRH)](https://www.gdatasoftware.com/blog/2
...
+There are multiple options to obtain the *TypeRefHasher*.
 
-1. `trh` -> Linux x64
-2. `trh.exe` -> Windows x64
+### Windows
+
+Install the tool with [winget](https://github.com/microsoft/winget-cli). The tool gets added to your `PATH` environment variable.
+
+```powershell
+winget install GDATA.TypeRefHasher
```
Comment (pkemkes) [inline README.md:18]: nit: I suppose this can only be tested once this PR is merged?

**Review by pkemkes — APPROVED** [review]
Looks good to me, but I should note that I am seeing most of this winget stuff for the first time.

File: README.md (hunk 2 above)
Comment (secana) [reply README.md:18]: Yeah, that's the case. I would wait the merge until the tool is on _winget_ such that we can try it at least once. Seems that this takes a lot of time as the PR is already open for a few days and still not merged by MS.

**Pull request merged**